probe_scraper/glean_checks.py

```diff
@@ -126,3 +127,83 @@ def check_for_duplicate_metrics(repositories, metrics_by_repo, emails):
         }
 
     return found_duplicates
+
+
+EXPIRED_METRICS_EMAIL_TEMPLATE = """
+The following metrics in {repo_name} will expire in the next {expire_days} days or have already expired.
+
+{expired_metrics}
+
+What to do about this:
+
+1. If the metric is no longer needed, remove it from the `metrics.yaml` file.
```

Comment:
Do we have *any* way to provide a link to the specific `metrics.yaml`?

Reply:
We could figure it out from what we have, if it's on Github (which everything is right now, even for m-c since we go through a github mirror).